packages/actions/src/Action.php

```diff
@@ -434,6 +434,25 @@ protected function getJsClickHandler(): ?string
             $argumentsParameter .= Js::from($arguments);
         }
 
+        $contextParameter = '';
+
+        if (count($context = $this->getContext())) {
+            $contextParameter .= ', ';
+            $contextParameter .= Js::from($context);
+
+            if ($argumentsParameter === '') {
+                $argumentsParameter = ', {}';
+            }
+        }
+
+        return "mountAction('{$this->getName()}'{$argumentsParameter}{$contextParameter})";
+    }
+
+    /**
+     * @return array<string, string|true>
```

Comment:
Strictly speaking we can do with `array<string, string|true>`, but for future flexibility `array<string, mixed>` is also a fine option.